prediction_market_agent_tooling/agent/development_tools/prophet_agent_tester.py

```
@@ -0,0 +1,252 @@
+    ) -> list[ProphetTestResult]:
+        filtered_dataset = dataset[dataset["agent_name"] == self.mocked_agent_name]
+        available_trades = len(filtered_dataset)
+        trades_to_process = min(self.max_trades_to_test_on, available_trades)
+
+        logger.info(
+            f"Found {available_trades} trades for {self.mocked_agent_name}, processing {trades_to_process}"
+        )
+
+        results = []
+
+        for index, (row_index, item) in enumerate(
```

Ruff (0.11.9)

Loop control variable `row_index` not used within loop body

Rename unused `row_index` to `_row_index`

(B007)